wp-content/themes/citylimits/less/_navbar.less

```diff
@@ -290,4 +290,53 @@ html.nav-open body {
     @media (min-width: 769px) {
         display: none;
     }
+}
+#secondary-nav {
+    .secondary-special-projects-nav {
+        list-style: none;
+        margin-left: 0;
+        margin-top: -5px;
+        display: flex;
+        align-items: center;
```

Comment:
If entries with different numbers of lines don't all have images, then the border-right is of different heights:

![Screen Shot 2019-10-22 at 18 22 12 ](https://user-images.githubusercontent.com/1754187/67339306-ee625800-f4f8-11e9-8eff-489ba363982d.png)

Do we care if that is the case, or is the operating assumption that they'll all have images?

Reply:
I _think_ the assumption is that they'll all have images, but I'll update it to make sure all of the list items get the same height so their border heights are the same.

Reply:
@benlk https://github.com/INN/umbrella-citylimits/pull/78/commits/5f9627bc08354b28f70805d6edc58164b0cb1eaa updates it so the list items are all the same height, giving them all the same height border